llvm/test/MC/Xtensa/Options/code_density-invalid.s

```diff
@@ -0,0 +1,17 @@
+# RUN: not llvm-mc -triple xtensa --mattr=+density %s 2>&1 | FileCheck %s
+
```

Comment:
I don't see the point of having an "Options" subdirectory

Reply:
I'm sorry, you mean that we don't need to create specific directory for Xtensa architecture options(extensions) tests in "../MC/Xtensa/"? Or use other directory structure, for example for each option(extension) create correspondence directory "../MC/Xtensa/CodeDensity"?

Reply:
I mean just leave it in test/MC/Xtensa

Reply:
I moved tests to MC/Xtensa from Option directory.